cpp/tests/strings/array_tests.cpp

```diff
@@ -47,8 +47,8 @@ TEST_F(StringsColumnTest, Sort)
 
 TEST_F(StringsColumnTest, SortZeroSizeStringsColumn)
 {
-  cudf::column_view zero_size_strings_column(
-    cudf::data_type{cudf::type_id::STRING}, 0, nullptr, nullptr, 0);
+  auto const zero_size_strings_column = cudf::make_empty_column(cudf::type_id::STRING)->view();
```

Comment:
is this safe? we're keeping a view of a column that's a temporary object. The pattern in `[gather_str_tests.cpp](https://github.com/rapidsai/cudf/pull/14030/files#diff-8c7735ba97a898ebe6d635f6606af58136f7e266a4aad7594b29c4ae18b055c7)` looks more intuitive to me.

Reply:
This is indeed safe. The return `column_view` contains no data. It has member data like type, etc. But the size and other values are set to null/zero. So there is no dependency on the column it came from.